### ISSUE TITLE
Removing the no_factors option on tiling

### ISSUE THREAD

**Issue by enadeau** [PermutaTriangle/Tilings#149]
I think it makes no sense to have a `no_factor` option on the monotone tree verification strategy. In fact in the current state the enumeration algorithm won't say a tiling is verified if it has more than one factor. I think if we have a monotone "forest" we should let the local verification handle that.
 
If we agree on that then the pack for this strategy should be focused on pulling out monotone leaves using interleaving factor.

**Comment by christianbean** [comment]
I agree with this, it is on my todo list to implement the enumeration interleaving factors.

**Comment by christianbean** [comment]
Now interleaving factors has the enumeration code implemented, we should be able to create a strategy pack that always finds a specification for such Tilings. We should also consider how to write down equations for whatever pack is made.

**Comment by enadeau** [comment]
We now have everything to build a pack for monotone tree tilings
